site/src/analytics-version-history.ts

```diff
@@ -0,0 +1,121 @@
+import { Checkbox } from "@material/mwc-checkbox";
+import {
+  css,
+  customElement,
+  html,
+  internalProperty,
+  LitElement,
+  property,
+} from "lit-element";
+import { AnalyticsDataHistory } from "../../worker/src/data";
+import "./components/analytics-chart";
+
+@customElement("analytics-version-history")
+export class AnalyticsVersionHistory extends LitElement {
+  @property({ attribute: false }) public historyData?: AnalyticsDataHistory[];
+
+  @property({ type: Boolean }) public isMobile = false;
+
+  @property({ type: Boolean }) public isDarkMode = false;
+
+  @internalProperty() private _logScale = false;
+
+  render() {
+    if (this.historyData === undefined) {
+      return html``;
+    }
+
+    const allVersions: Set<string> = new Set();
+    this.historyData.forEach((entry) => {
+      if (entry.versions) {
+        Object.keys(entry.versions).forEach((version) => allVersions.add(version));
+      }
+    });
+
```

Comment:
```suggestion

    if (allVersions.size === 0) {
      return html``;
    }

```

This will only be true the first hour after release, but we should still guard against it.

Reply:
I wonder.... should we make this > 2 instead?
<https://deploy-preview-219--home-assistant-analytics.netlify.app/> with only 1 set we only get dots, with at least 2 there will be a line

Reply:
Tbh I don't think it would really matter that much, but I updated it anyway. It doesn't hurt either.

Reply:
It did not work anyway

Reply:
Think I got it now with the latest changes. Did some small optimization along the way, too.